packages/blockchain-wallet-v4/src/redux/payment/eth/sagas.js

```diff
@@ -53,9 +53,9 @@ export default ({ api }) => {
         let unconfirmedTx = false
         if (latestTx) {
           const ethOptionsR = yield select(S.walletOptions.getEthereumTxFuse)
-          const lastTxFuse = ethOptionsR.getOrElse(86400000)
+          const lastTxFuse = ethOptionsR.getOrElse(86400) * 1000
```

Comment:
This will give you 86400000 * 1000

https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/1e39c1728792a95d60183858d30e5148765b0b2e/config/wallet-options-v4.json#L75